javascript/javascript1/week2/classwork/countby-sound.js

```diff
@@ -0,0 +1,26 @@
+ const names = [
+  "Chris",
+  "Anne",
+  "Colin",
+  "Terri",
+  "Phil",
+  "Lola",
+  "Sam",
+  "Kay",
+  "Bruce",
+];
+
+for (i = 0; i < names.length; i++) {
+  if (names[i].toLowerCase().includes("a") === true) continue;
```

Comment:
Includes already returns a boolean, I think the comparison can be accomplished without the check with ===true